### PR TITLE
Feature branch 

### PR DESCRIPTION
**Pull request by pacyL2K19**
#  The aim of this PR 
This PR contains : 
- User model, controller, and routes 
- Forms to create and edit user information
- Code linters

### CODE REVIEW

**Review by LamiaSristy — COMMENTED** [review]
# Status: Invalid Review Request ✖️

Hello @pacyL2K19 and @RailonA, I can see lots of efforts here, but unfortunately, I have to make this project invalid.

## Linters checks are not successful.
**Please add and fix the linter errors so that your PR is professional, all checks pass and the code follows the best practices for the using techniques. By following the linter recommendations you are ensuring your code is more readable, modern, and less bug-prone.**

**Kindly take a step back and configure [linters](https://github.com/microverseinc/linters-config/tree/master/ror) correctly in your project**

**I have to mark this review request as invalid because none of your GitHub Actions checks were successful**

This Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready!

Feel free to reach me on slack @lamiasristy 🙋

Keep up the spirit always. Happy Coding 💻

**Comment by pacyL2K19** [comment]
Thank you for your review, it was helpful
@LamiaSristy

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -0,0 +1,36 @@
+class UsersController < ApplicationController
+  def new
+    @user = User.new
+  end
+
+  def create
+    # @user = User.new(username: params[:username], email: params[:email], password: params[:password])
```
Comment (lekegitrepo) [inline app/controllers/users_controller.rb]: - Remove the commented code

**Review by lekegitrepo — CHANGES_REQUESTED** [review]
## Status: Changes Required! :recycle:
### Hi @pacyL2K19 and @RailonA :clap: :clap:
#### This is a good project :muscle: and it's well implemented and the readme is well detailed :writing_hand:
#### However, some requirements are missing/not implemented correctly. Check my comments below :point_down:

#### Implementation:
 - **user model:** should have fields: `:username`, `:email `and `:password`. The `:password` field is missing and when added don't forget to add validation.

#### Optional:
 - You can create partial for user form and reuse in edit and new view

#### _Reviewed by Temitayo Adeleke_

**Review by oloomoses — APPROVED** [review]
# Congratulations, Project Approved. :muscle: :tada: :clap: 
**Great job working on the suggested changes from the previous review.
You have passed all the requirements for this project :muscle: 
Feel free to merge and close this pull request.**


**I wish you the best in your next project and see you at the top :rocket:** 
#### Happy Coding
_Slack: Oloo Moses_

**Pull request merged**